Code/Tools/ProjectManager/Source/ProjectUtils.cpp

```diff
@@ -119,36 +229,63 @@ namespace O3DE::ProjectManager
                     return false;
                 }
 
-                QGuiApplication::setOverrideCursor(QCursor(Qt::WaitCursor));
-                copyResult = CopyProject(origPath, newPath);
-                QGuiApplication::restoreOverrideCursor();
-
+                copyResult = CopyProject(origPath, newPath, parent);
             }
 
             return copyResult;
         }
 
-        bool CopyProject(const QString& origPath, const QString& newPath)
+        bool CopyProject(const QString& origPath, const QString& newPath, QWidget* parent)
```

Comment:
Can you add a parameter option to suppress dialogs?

Reply:
Why would we want that? Especially without an option for the user to actually enable that supression - let's get this in once we have a settings window or something similar, but I would argue that even such a setting doesn't provide any user benefit.

Reply:
I was thinking for the purpose of automated tests. We don't want them popping up dialogs because it makes the automation more awkward.